.github/workflows/build-and-test-orchard-core.yml

```diff
@@ -213,6 +213,10 @@ jobs:
           binary-log-artifact-retention-days: ${{ inputs.build-binary-log-artifact-retention-days }}
           warnings-as-errors: ${{ inputs.warnings-as-errors }}
 
+      - name: Print configuration summary
```

Comment:
Is this really generally necessary for every build? Because if not, then have it configurable, disabled by default, and only enabled for OSOCE:

Reply:
As I said above (https://github.com/Lombiq/GitHub-Actions/pull/223#discussion_r1157306921), the NE builds have been failing in different projects, and it would be cumbersome to have them all update their workflow configuration right now.

This is still work in progress, and I'm trying to gather the version data to see if the used versions of Node.js and PNPM are connected to the build errors.

Once we find out more and won't need this by default, we can definitely turn it off. But not at the moment.

Reply:
OK